demo/gallery/bootstrap.js

```diff
@@ -36,6 +36,33 @@ function build(E, log) {
     const aliceP = E(aliceMaker).make(gallery.userFacet);
     await E(aliceP).doSellAndBuy();
   }
+  async function testAliceSellsToBob(aliceMaker, bobMaker, gallery, handoff) {
+    log('starting testAliceSellsToBob');
+    const { userFacet, adminFacet } = gallery;
+    const aliceP = E(aliceMaker).make(userFacet);
+    const bobP = E(bobMaker).make(userFacet);
+    const { dustIssuer } = userFacet.getIssuers();
+    const aliceDustPurseP = dustIssuer.makeEmptyPurse('Alice dust purse');
+    const bobDust = adminFacet.dustMint.mint(37, 'bob purse');
```

Comment:
I think that both alice's dust purse and bob's dust purse are the same type, and (I could be wrong on this) but neither are presences or promises.

Reply:
I think you're right that the gallery, and thus the issuers are local to the bootstrap vat. 

Are you suggesting that since they're from the same issuer, the same facet could make both purses? makeEmptyPurse() and mint() are only available from separate facets. Would it be better to use mint(0, ...) to make Alice's purse?

Reply:
Oh, I was talking about the 'P' suffix and just saying that it seems like they should either both have it or not have it. I think in this case there shouldn't be a P suffix.